pages/cadastro/index.public.js

```diff
@@ -181,6 +181,15 @@ function SignUpForm() {
             aria-label="Criar cadastro">
             Criar cadastro
           </Button>
+          <div style={{ textAlign: 'center', width: '100%' }}>
+            <Text sx={{ fontWeight: 'bold', fontSize: '12px', color: '#a6a6a6' }}>
+              Ao criar uma conta, você estará concordando com os{' '}
```

Comment:
Gostei da frase, mas gostaria de saber a opinião da Turma e, se for o caso, analisar outras alternativas.